src/models/user.server.model.js

```diff
@@ -26,9 +26,11 @@ const userSchema = new Schema({
     },
     createdBy: {type: Schema.ObjectId, ref: 'User', required: true},
     createdOn: {type: Date, default: Date.now}
-}, {collection: 'sp_user'});
+}, {
+    collection: 'sp_user'
+});
 
-userSchema.virtual('fullName').get(() => {
+userSchema.virtual('fullName').get(function () {
```

Comment:
The value of `this` cannot be bound to the local scope. Therefore, using regular functions is needed (instead of arrow functions) to configure virtual properties in our models. In detail, inside virtual properties, `this` was returning `undefined` using arrow functions. The value of `this` needs to be bound at the parent scope.